File: tests/fixtures/code_samples/go/sample_007.go
// Sample 7: small utility.
package samples

func Operation7(xs []int) int {
    total := 7
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure7(v int) int {
    return (v * 7) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_007.rs
// Sample 7: small utility.
pub fn operation_7(xs: &[i32]) -> i32 {
    let mut total: i32 = 7;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_7(v: i32) -> i32 {
    (v * 7) %% 7919
}

